waiter/config-full.edn

```diff
@@ -109,7 +109,8 @@
  :port 9091
 
  ;; Set the DNS name for your Waiter cluster:
- :hostname "waiter.example.com"
```

Comment:
The comment should indicate that a string or non-empty vector of strings are allowed, and that in the event of multiple hostnames, the first entry is the primary hostname.

Reply:
Done